ddtrace/internal/native/_native.pyi

```diff
@@ -447,11 +448,58 @@ class SerializationError(Exception):
 
     ...
 
-def ffande_process_config(config_bytes: bytes) -> Optional[bool]:
+class ffe:
     """
-    Process feature flagging and experimentation configuration rules.
-
-    :param config_bytes: Raw bytes containing the configuration data
-    :return: True if processing was successful, False otherwise, None on error
+    Native Feature Flags and Experimentation module.
     """
-    ...
+
+    class FlagType(Enum):
+        String = ...
+        Integer = ...
+        Float = ...
+        Boolean = ...
+        Object = ...
+
+    class Reason(Enum):
+        Static = ...
+        Default = ...
+        TargetingMatch = ...
+        Split = ...
+        Cached = ...
+        Disabled = ...
+        Unknown = ...
+        Stale = ...
+        Error = ...
+
+    class ErrorCode(Enum):
+        TypeMismatch = ...
+        ParseError = ...
+        FlagNotFound = ...
+        TargetingKeyMissing = ...
+        InvalidContext = ...
+        ProviderNotReady = ...
+        General = ...
```

Comment:
No `ProviderFatal` because OF handles that? https://github.com/open-feature/python-sdk/blob/4d0779c56cc66d3fd9754a74f45c5c0a8e01c8ee/openfeature/exception.py#L168-L175